source/_docs/guides/drupal-8-composer-no-ci.md

```diff
@@ -0,0 +1,240 @@
+      <h3 class="info panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-wrench"></span>Remove Test Suites</h3>
+    </a>
+  </div>
+  <div id="unique-anchor" class="collapse" markdown="1" style="padding:10px;">
+    First, delete the `tests` directory. Next, you will need to modify `composer.json`.
+
+    * Remove all dependencies in the `require-dev` section
+    * Update the `scripts` section to the following:
+
+    ```
+        "scripts": {
+            "build-assets": [
+                "@prepare-for-pantheon",
+                "composer install --optimize-autoloader --no-dev"
+            ],
+            "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
+            "prepare-for-pantheon": "DrupalProject\\composer\\ScriptHandler::prepareForPantheon",
+            "post-install-cmd": [
+                "@drupal-scaffold",
+                "DrupalProject\\composer\\ScriptHandler::createRequiredFiles"
+            ],
+            "post-update-cmd": [
+                "DrupalProject\\composer\\ScriptHandler::createRequiredFiles"
+            ],
+            "post-create-project-cmd": [
+                "@drupal-scaffold",
+                "DrupalProject\\composer\\ScriptHandler::createRequiredFiles"
+            ]
+        },
+    ```
+  </div>
+</div>
+
+## Managing Drupal with Composer
+
+### Downloading Drupal Dependencies with Composer
+
+Normally the next step would be going through the standard Drupal installation. But since we’re using Composer, none of the core files exist yet. Let’s use Composer to install Drupal core.
+
+Since we modified `composer.json` we will need to update Composer to download the new dependencies. 
+
+```
+composer update
+```
+
+This may take a while as all of Drupal core and its dependencies will be downloaded. Subsequent updates should take less time.
+
+![image of terminal running a composer install](/source/docs/assets/images/guides/drupal-8-composer-no-ci/drops-8-composer-install.png)
+
+```
+git status
+```
+
+**Important:** The `example-drops-8-composer` `.gitignore` file assumes that you’re using a build step with continuous integration. To make it compatible with this method we need to remove everything above the CUT section of the `.gitignore` file. Without this modification critical components, such as Drupal core and contrib modules, will be ignored and not pushed to Pantheon. Now let’s run git status again to make sure everything is included.
+
+```
+git status
+```
+
+![image of git status showing the changed files in red](/source/docs/assets/images/guides/drupal-8-composer-no-ci/drops-8-composer-git-status-after-installing-d8.png)
+
+Set the site to git mode.
+
+```
+terminus connection:set $PANTHEON_SITE_NAME.dev git
+```
+
+Add and commit the code files. A git force push is necessary initially but subsequent pushes should not need `--force`.
```

Comment:
Let's add an explanation for why. 

> A git force push is necessary because we are writing over the empty repository on Pantheon with our new history that was started on the local machine. Subsequent pushes should not use ` --force`.